bokehjs/src/coffee/model.coffee

```diff
@@ -17,6 +21,42 @@ export class Model extends HasProps
       for cb in callbacks
         @listenTo(@, evt, () -> cb.execute(@))
 
+    @listenTo(@, 'change:js_event_callbacks', () -> @_update_event_callbacks)
+
+
+  _process_event : (event) ->
+    # Given an applicable event, execute the associated callback
+    if not event.constructor.event_name
+       logger.warn('Event is still passed as a string: ' + event)
+       return
+    if not event.constructor.applicable_models.some((m) => m == @.type)
```

Comment:
I'm speaking about `m == @.type`. `m` and `@` are models, so this should use `instanceof` check.